app/src/main/res/values-ko/localized.xml

```diff
@@ -111,9 +111,9 @@
     <string name="p_debug_mode">디버그 모드</string>
     <string name="p_debug_mode_summary">대조되고 있는 부분 표시하기</string>
     <string name="p_ignore_notch">노치 계산 무시</string>
-    <string name="p_ignore_notch_summary">삼성 폰에서 유용</string>
+    <string name="p_ignore_notch_summary">삼성 폰에서 유용 (S10 이상)</string>
```

Comment:
Don't put S10 there. It varies depending on when your phone receives updates from Samsung.

Reply:
Oh okay. A lot of people keeps turning on the notch settings just because it says it's useful for samsung phones. Will fix.

Reply:
Common masters are mistaken for having to turn on this option on "All" Samsung phones.

Reply:
That has been clarified so don't worry